barrantes/index.html

```diff
@@ -0,0 +1,176 @@
+                <h2>Testimonials</h2>
+                <ul>
+                    <li>This webpage stands out with its clean design and 
+                        well-organized content. Good job! - Eliza
+                    </li>
+                    <li>The use of engaging visuals and interactive elements on 
+                        the website makes it visually appealing and engaging. 
+                        - Mikaella
+                    </li>
+                    <li>You're creative genius knows no bounds, and they have a 
+                        rare ability to think divergently and connect seemingly 
+                        the best ideas.-Precious
+                    </li>
+                    <li>Nice appearance on your webpage! Good job!- Alona</li>
+                </ul>
+            </div>
+        </div>
+        <div class="section-container">
+            <div class="slide">
+                <h2>Thoughts about Online Class</h2>
+                <p>Online classes offer benefits during unexpected events or 
+                    scenarios, but they should not be the sole choice. For 
+                    hands-on subjects, f2f classes are preferred due to 
+                    professor-student interaction and easy communication. Online 
+                    classes are a preferred choice for suddent situations but, 
+                    f2f classes has its own benefits especially in this type of 
+                    course or program.
+                </p>
+            </div>
+        </div>
+    </div>
+    <div class="gallery">
+        <h2>Gallery</h2>
+        <div class="gallery-item">
+            <img src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcRB0UQBJTUYLyHaIv-cDA7kWbT6AjBCZIgByKiwXkrJig&s" />
+        </div>
+        <div class="gallery-item">
+            <img src="https://i.pinimg.com/736x/0f/21/7d/0f217d0189f841ae794500966ab1845a.jpg" />
+        </div>
+        <div class="gallery-item">
+            <img src="https://i.pinimg.com/564x/ac/f5/13/acf513e438c3be05d8694ff1bb6f4712.jpg" />
+        </div>
+        <div class="gallery-item">
+            <img src="https://i.pinimg.com/736x/52/b4/c5/52b4c5f65a92e2167a89a0621bdc2a8e.jpg" />
+        </div>
+        <div class="gallery-item">
+            <img src="https://static.independent.co.uk/s3fs-public/thumbnails/image/2015/06/10/10/jCubeDog.jpg?width=1200" />
+        </div>
+        <div class="gallery-item">
+            <img src="https://i.pinimg.com/564x/5d/29/93/5d2993b3ad0f1d37c630d3132114a1c0.jpg" />
+        </div>
+    </div>
+    <div class="contact">
+        <h2>Contact Me</h2>
+        <form>
+            <div class="email">
+                <label for="email">Email:</label>
+                <input type="email" id="email" name="email" />
+            </div>
+            <div class="name-contact">
+                <label for="Name">Full Name:</label>
+                <input type="text" id="name-contact" name="Name" />
```

Comment:
IDs should be written in snake case. All lower case separated by underscore. For reference, please click on the link provided below to access further information.

```suggestion
                <input type="text" id="name_contact" name="Name" />
```


[https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx- TPQxdLySow/edit#slide=id.g24da8de92d7_0_5](https://github.com/PUP-BSIT/ex16-tech_nx/pull/1/commits/url)